File: backend/src/main/kotlin/tiago/canilhas/foobs/domain/Food.kt
package tiago.canilhas.foobs.domain

data class Food (
    val id: Int,
    val name: String,
    val brand: String?,
    val protein: Double,
    val carbohydrate: Double,
    val fat: Double,
    val fiber: Double,
    val calories: Double,
    val units: List<Unit> = emptyList(),
)

File: frontend/src/app/shared/models/food.ts
import { FoodUnit } from './food-unit';

export interface Food {
    id: number;
    name: string;
    brand?: string;
    protein: number;
    carbohydrate: number;
    fat: number;
    fiber: number;
    calories: number;
    units: FoodUnit[];
}
